### DISCUSSION TITLE
Integrate with Android notifications for media playback and phone calls

### DISCUSSION THREAD

**Discussion by TobiPeterG** [osmandapp/OsmAnd#20914] [Ideas]
It would be great if I could add a widget to Pause/skip etc. The current music I'm listening to while driving.
It would also be great if OSMAnd allowed me to see any calls I get and accept them and stop them again without leaving the app.

**Comment by sjvudp** [comment]
I think media and call management should not be part of the app, but obviously the app should silence voice output during a phone call.
I could also "auto-duck" the music playing when making voice announcements.
A decent media player on Android would allow you to pull down a menu whee you can pause/resume and skip forward/back, so OsmAnd does not have to provide that. Android Auto and headsets also provide such functions. Likewise for incoming calls.
